clients.json

```
@@ -775,7 +775,17 @@
     "authors": [],
     "active": true
   },
-
+  
+  {
+    "name": "xredis",
+    "language": "Node.js",
+    "repository": "https://github.com/razaellahi/xredis",
+    "description": "Redis client with redis ACL features",
+    "authors": ["RazaEllai"],
```

This is supposed to be the author's Twitter account - if you don't have one just leave it blank like so:
```suggestion
    "authors": [],
```